app/src/org/commcare/dalvik/activities/CommCareVerificationActivity.java

```diff
@@ -227,6 +251,20 @@ public CustomProgressDialog generateProgressDialog(int taskId) {
         return null;
     }
 
+    @Override
+    public void onClick(View v) {
+        switch (v.getId()) {
+        case R.id.skip_verification_button:
+            Intent i = new Intent(getIntent());
+            setResult(RESULT_CANCELED, i);
+            finish();
+            break;
+        case R.id.screen_multimedia_retry:
+            verifyResourceInstall();

```

Comment:
Please indent switch statements such that 'case' is indented 4 spaces more than 'switch'.